pkgs/desktops/gnome/core/mutter/default.nix

```diff
@@ -65,6 +65,13 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://gitlab.gnome.org/GNOME/mutter/-/commit/285a5a4d54ca83b136b787ce5ebf1d774f9499d5.patch";
       sha256 = "/npUE3idMSTVlFptsDpZmGWjZ/d2gqruVlJKq4eF4xU=";
     })
+
+    # Fix focus regression.
```

Comment:
Unfortunately looks like there is more than one "focus regression": https://gitlab.gnome.org/GNOME/mutter/-/issues/2626. Can someone try this PR and decide how bad the issue is? I wonder if I should revert the series of "Do not update focus" commits or even revert the whole mutter bump.